### PR TITLE
Create toolbar and headline section

### PR DESCRIPTION
**Pull request by wayungi**
### This PR adds 
 - a background image for the page
 - a toolbar with the logo and navigation icon
 - a background image for the headline section, social media icons, introduction and profile messages

### CODE REVIEW

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @wayungi ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Check the comments under the review for more suggestions. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Github flow used properly.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

- [ ] The commit message should be written in the imperative tense and the first letter will be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details.](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [ ] For readability, please add two spaces of indentation. Do not use the tab key. You did it throughout your code in the `HTML` file. Please fix it. [For the HTML & CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)we should follow the rules.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css [hunk 1]
```diff
@@ -93,44 +93,79 @@ div {
 }
 
 a.connect {
-  padding: 5px;
-  border: 1px solid #7F8CFF;
+  /* padding: 5px; */
+  /* border: 1px solid #7F8CFF; */
```
Comment (xtrahuman) [inline style.css]: Kindly remove all commented unused code from your repo

File: index.html [hunk 2]
```diff
@@ -14,16 +14,16 @@
 
     <div class="page-wrappr">
         <header class="header-section">
-            <article class="logo-placeholder">
-                <h1 class="logo">My Logo</h1>
-            </article>
+            <div class="logo-placeholder">
+                <p class="logo">My Logo</p>
```
Comment (xtrahuman) [inline index.html:18]: It is best practice to use `<a>` for your logo, Kindly change from p to a tag

**Review by xtrahuman — CHANGES_REQUESTED** [review]
Hello @wayungi, 

Amazing job so far, Keep up the positive energy 👍 😄 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

image

 ## Required Changes ♻️
_Kindly Check the comments under the review._ 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — APPROVED** [review]
Hi @wayungi,
# Status: Approved ✔️ ✔️ ✔️ 
 
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6d43496a436773336e575157664a5a7650412f67697068792d646f776e73697a65642d6c617267652e676966


Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**